package.json

```diff
@@ -10,9 +10,13 @@
     "prettier": "prettier --write ."
   },
   "dependencies": {
+    "formik": "^2.2.9",
     "next": "13.2.4",
+    "next-auth": "^4.20.1",
```

Comment:
question(block): Why you choose `next-auth`?

Reply:
One justification for choosing NextAuth for our project is its ease of implementation. While there were no specific requirements that necessitated the use of NextAuth. Do you have any other recommendations for authentication and authorization libraries I should consider?

Reply:
I would use the Context API, but if Next Auth solves it for you to register, login and access via Google, that's fine, but you'll have to hit the backend without using the ready next auth providers, I don't know how this lib works behaves with custom providers outside the library's own domain